modules/ocf/manifests/configbuilder.pp

```diff
@@ -0,0 +1,22 @@
+define ocf::configbuilder(
```

Comment:
Maybe `config_builder` instead given that's the convention we (generally) use for other class names (underscores to separate words).

We do have some exceptions like `ocf::hostkeys`, `ocf::autologout`, or `ocf::crondeny`, but our newer stuff is generally named with underscore separators (like manifests in `ocf_mesos`, `ocf::firewall`, and `ocf::ssl`)